docker/scripts/entrypoint.sh

```diff
@@ -6,23 +6,23 @@
 # - preparation before the node start (example keys injection)
 # - launch of the actual node
 # 
-# Environment variables should generally be in the form `NH_*`
```

Comment:
Being the renaming from `NH_/NH_CONF` to `ZKV_/ZKV_CONF` a breaking change, let's make sure we align the `compose` projects accordingly.

Reply:
Will merge this one and start adjusting also the other repos :)